### ISSUE TITLE
About YOLOv3, 4

### ISSUE THREAD

**Issue by tuan3ihust** [Xilinx/Vitis-AI#1097]
I trained my model on Darknet. Should I convert my model to TensorFlow or Caffe (which is not supported on Vitis AI 2.x), which is better?

**Comment by wangxd-xlnx** [comment]
Hi @tuanmonkeyy 

Vitis-AI 3.0 has supported YOLOv5 pytorch. Please find it in model_zoo. But for yolov5/v6 series model, we just provided a README document to share the accuracy and performance instead of the model itself due to limitations of the original model license.

If you want to get this model, please refer to the contact information in README and contact MKT for further support.
